### ISSUE TITLE
About val.txt

### ISSUE THREAD

**Issue by WindVChen** [PeiqinZhuang/API-Net#8]
Hi,

I recently did some research on fine-grained classification, and I found that there is a 'val.txt' in your code. Is the 'val.txt' in the code actually the test set in datasets like CUB? Whether should I split trainset into train and val set when I do some experiments on datasets like CUB_200_2011, since I also didn't find any description about validation set in other existing papers?

Thank you.

Kind regards.
